client/signup/reskinned-processing-screen/index.jsx

```diff
@@ -19,28 +18,30 @@ const useSteps = ( { flowName, hasPaidDomain, isDestinationSetupSiteFlow } ) =>
 
 	switch ( flowName ) {
 		case 'launch-site':
-			steps = [ __( 'Your site will be live shortly.' ) ]; // copy from 'packages/launch/src/focused-launch/success'
+			steps = [ { title: __( 'Your site will be live shortly.' ) } ]; // copy from 'packages/launch/src/focused-launch/success'
 			break;
 		default:
 			steps = [
-				! isDestinationSetupSiteFlow && __( 'Building your site' ),
-				hasPaidDomain && __( 'Getting your domain' ),
-				! isDestinationSetupSiteFlow && __( 'Applying design' ),
+				! isDestinationSetupSiteFlow && { title: __( 'Building your site' ) },
+				hasPaidDomain && { title: __( 'Getting your domain' ) },
+				! isDestinationSetupSiteFlow && { title: __( 'Applying design' ) },
 			];
 	}
 
 	if ( flowsWithDesignPicker.includes( flowName ) ) {
+		// Custom durations give a more believable loading effect while setting up
+		// the site with headstart. Which can take quite a long time.
 		steps = [
-			__( 'Laying the foundations' ),
-			__( 'Turning on the lights' ),
-			__( 'Making it beautiful' ),
-			__( 'Personalizing your site' ),
-			__( 'Sprinkling some magic' ),
-			__( 'Securing your data' ),
-			__( 'Enabling encryption' ),
-			__( 'Optimizing your content' ),
-			__( 'Applying a shiny top coat' ),
-			__( 'Closing the loop' ),
+			{ title: __( 'Laying the foundations' ), duration: 7000 },
+			{ title: __( 'Turning on the lights' ), duration: 3000 },
+			{ title: __( 'Making it beautiful' ), duration: 4000 },
+			{ title: __( 'Personalizing your site' ), duration: 7000 },
+			{ title: __( 'Sprinkling some magic' ), duration: 4000 },
+			{ title: __( 'Securing your data' ), duration: 9000 },
+			{ title: __( 'Enabling encryption' ), duration: 3000 },
+			{ title: __( 'Optimizing your content' ), duration: 6000 },
+			{ title: __( 'Applying a shiny top coat' ), duration: 4000 },
+			{ title: __( 'Closing the loop' ) },
```

Comment:
I think we should also give the duration to this step?

Reply:
The last step was just supposed to stay focused until the processing completes, so I don't think we should add a duration here. https://github.com/Automattic/wp-calypso/issues/56831#issuecomment-948099230

Reply:
Oops...I missed that